esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -673,7 +674,31 @@ define([
                             node.innerText = Utility.valueCleanUp(value);
                         },
                     },
-                    Modified: { label: this.i18n.ModifiedUTCGMT, width: 162 }
+                    MinSkew: {
+                        label: nlsHPCC.MinSkew, width: 60,
+                        formatter: function (value, row) {
+                            return value !== undefined ? value : "";
+                        }
+                    },
+                    MaxSkew: {
+                        label: nlsHPCC.MaxSkew, width: 60,
+                        formatter: function (value, row) {
+                            return value !== undefined ? value : "";
+                        }
+                    },
+                    Modified: { label: this.i18n.ModifiedUTCGMT, width: 162 },
+                    AtRestCost: {
```

Comment:
I missed adding these fields to the dojo component in HPCC-27149, so just thought I'd add them now.